src/Compilers/CSharp/Portable/Symbols/LexicalSortKey.cs

```diff
@@ -53,8 +53,7 @@ public int Position
         // until later when it is known if they can be optimized or not.
         // As a result the last emitted method tokens are synthetic ctor and then synthetic cctor (if not optimized)
         // Since it is not too hard, we will try keeping the same order just to be easy on metadata diffing tools and such.
-        public static readonly LexicalSortKey SynthesizedRecordCopyCtor = new LexicalSortKey() { _treeOrdinal = int.MaxValue, _position = int.MaxValue - 3 };
-        public static readonly LexicalSortKey SynthesizedRecordCtor = new LexicalSortKey() { _treeOrdinal = int.MaxValue, _position = int.MaxValue - 2 };
+        public static readonly LexicalSortKey SynthesizedRecordCopyCtor = new LexicalSortKey() { _treeOrdinal = int.MaxValue, _position = int.MaxValue - 2 };
```

Comment:
should the above SortKey positions be adjusted?

Reply:
I've replaced this code in https://github.com/dotnet/roslyn/pull/44882.

Reply:
> should the above SortKey positions be adjusted?

I am not following. Could you elaborate please?

---
In reply to: [436055537](https://github.com/dotnet/roslyn/pull/44876#discussion_r436055537) [](ancestors = 436055537)

Reply:
For instance SynthesizedRecordObjEquals still has `_position = int.MaxValue - 4` while there is no symbol with position `int.MaxValue - 3`. I doubt this negatively affects anything so perhaps we can just leave it as is, especially if changing it may introduce more conflicts with other PRs.

Reply:
> I've replaced this code in [https://github.com/dotnet/roslyn/pull/44882](https://github.com/dotnet/roslyn/pull/44882).

Reverted to avoid a conflict

---
In reply to: [436071673](https://github.com/dotnet/roslyn/pull/44876#discussion_r436071673) [](ancestors = 436071673)

Reply:
> For instance SynthesizedRecordObjEquals still has _position = int.MaxValue - 4 while there is no symbol with position int.MaxValue - 3. I doubt this negatively affects anything so perhaps we can just leave it as is, especially if changing it may introduce more conflicts with other PRs.

I reverted the change in favor of in [https://github.com/dotnet/roslyn/pull/44882](https://github.com/dotnet/roslyn/pull/44882). 

---
In reply to: [436077962](https://github.com/dotnet/roslyn/pull/44876#discussion_r436077962) [](ancestors = 436077962)